documentation/overview/account_setup/registration-and-login.md

```diff
@@ -3,89 +3,57 @@ ospool:
   path: overview/account_setup/registration-and-login.md
 ---
 
-Registration and Login for OSG Connect 
+Registration and Login for OSPool Access
 ====================================
 
 
+The major steps to get started on the OSPool are: 
 
-## Registration and Login for OSG Connect
-
-The major steps to getting started on OSG Connect are: 
-
-* apply for an OSG Connect account
-* meet with an OSG Connect staff member for an short consultation and orientation. 
-* join and set your default "project"
-* upload `.ssh` keys to the OSG Connect website
+* apply for access to the OSPool
+* meet with a facilitation team member for an short consultation and orientation. 
+* register for a specific OSPool Access Point
+* log in to your designated Access Point
 
 Each of these is detailed in the guide below. 
+Once you've gone through these steps, you should be able to begin running work! 
 
-Once you've gone through these steps, you should be able to login to the OSG Connect
-submit node.  
-
-## Account Creation
-
-### Sign in to or create an account
-
-Start by creating an OSG Connect account. Visit the [OSG Connect web
-site](<https://osgconnect.net/>), then click on the **Sign Up** button. You 
-will need to agree to our Acceptable Use Policy in order to get to the 
-main log in screen. 
+## Apply for OSPool Access
 
-The main log in screen will prompt you to sign in via your primary institutional 
-affiliation. You'll be directed to a discovery
-service which asks you what your home institution is. (If you've used CILogon
-before it may already know your home institution and skip this step.) Locate
-your institution in the list, or type its name to find matches. 
+To start, fill out the interest form on this OSG Portal site: 
 
-> **No Institutional Identity?**
-> 
-> If you don't have an institutional identity or can't find your institution 
-> on the provided list, either (separately) sign up for a Globus ID and follow 
-> the link for that option on this page, or contact the OSG Connect support 
-> team for guidance at support@osg-htc.org
+[OS Pool Account Request](https://portal.osg-htc.org/application)
```

Comment:
So I apply for an `OS Pool Account` and then will get an `OSG Manage` account. Branding seems inconsistent.

Reply:
You're right. :P